### PR TITLE
Created top menu

### PR DESCRIPTION
**Pull request by SergioVed**
Created tom menu NavBar with array of elements end endpoint "/"

### CODE REVIEW

File: zoo-hope/src/components/navBar/index.tsx [hunk 1]
```diff
@@ -0,0 +1,72 @@
...
+import NavDropdown from "react-bootstrap/NavDropdown";
+import "../../styles/index.scss";
+import { useState } from "react";
+import LogoNavBar from "../../images/logo/logo.png"
+
+export const NavBar = () => {
+  const [clickedButtonUA, setClickedButtonUA] = useState(true);
+  const [clickedButtonEN, setClickedButtonEN] = useState(false);
+
+  const elements = [
```
Comment (Osttik) [inline zoo-hope/src/components/navBar/index.tsx]: This data will be better to move out of file to new folder like src/data/
and create nav.ts
And add there that
So use just import from nav.ts to your file

**Review by Osttik — CHANGES_REQUESTED** [review]

**Comment by Osttik** [comment]
Everything else is good

File: zoo-hope/src/components/navBar/index.tsx (hunk 1 above)
Comment (SergioVed) [reply zoo-hope/src/components/navBar/index.tsx]: done

File: zoo-hope/src/data/nav.ts [hunk 2]
```diff
@@ -0,0 +1,42 @@
...
+      ],
+    },
+    {
+      name: "Стерилізація",
+      Ielements: [
+        { name: "DropDownLink1", url: "/" },
+      ],
+    }]
+
+    const [clickedButtonUA, setClickedButtonUA] = useState(true);
```
Comment (Osttik) [inline zoo-hope/src/data/nav.ts]: This sections are not necessary to more out
return please useState back
Threre should be only that:
const elements = [...];
export elements;

**Review by Osttik — CHANGES_REQUESTED** [review]

File: zoo-hope/src/data/nav.ts (hunk 2 above)
Comment (SergioVed) [reply zoo-hope/src/data/nav.ts]: done

**Pull request merged**